Patterns/Japanese/Japanese-Numbers.yaml

```diff
@@ -194,6 +197,12 @@ DoubleAllFloatRegex: !nestedRegex
 DoubleExponentialNotationRegex: !nestedRegex
   def: (?<!{ZeroToNineFullHalfRegex}+[\.．])({NegativeNumberTermsRegexNum}\s*)?{ZeroToNineFullHalfRegex}+([\.．]{ZeroToNineFullHalfRegex}+)?e(([-－+＋]*[1-9１２３４５６７８９]{ZeroToNineFullHalfRegex}*)|[0０](?!{ZeroToNineFullHalfRegex}+))
   references: [ZeroToNineFullHalfRegex, NegativeNumberTermsRegexNum]
+DoubleExponentialNotationRegex1: !nestedRegex
+  def: (?<!{ZeroToNineFullHalfRegex}+[\.．])(({NegativeNumberTermsRegexNum}|{NegativeNumberTermsRegex})\s*)?{ZeroToNineFullHalfRegex}+([\.．]{ZeroToNineFullHalfRegex}+)?(×)?(10)?(e|の)(([-－+＋]*[1-9１２３４５６７８９]{ZeroToNineFullHalfRegex}*)[乗]|[0０](?!{ZeroToNineFullHalfRegex}+))
```

Comment:
Why do we need a separate regex from DoubleExponentialNotationRegex?
Also, aren't the full-width digits in both redundant (due to pre-processing)? Removing them would bring some performance benefit.

Reply:
@tellarin 
That makes sense. We did that for the exponential new cases. For now we are going to revert that particular regex DoubleExponentialNotationRegex1 from the PR. We'll revisit those cases and will fix that.